src/online2bin/online2-tcp-nnet3-decode-faster.cc

```diff
@@ -425,6 +449,26 @@ Vector<BaseFloat> TcpServer::getChunk() {
   return buf;
 }
 
+void TcpServer::SaveRemainder(int remainder) {
+  rem_.Resize(static_cast<MatrixIndexT>(remainder));
+
+  int32 offset = has_read_ - remainder;
+  for (size_t i = offset; i < has_read_; i++)
+    rem_(i - offset) = static_cast<BaseFloat>(samp_buf_[i]);
+}
+
+bool TcpServer::HasReaminder() {
```

Comment:
typo: HasReaminder -> HasRemainder

There are a few more occurrences of this in this file.

Reply:
Refactored.